java/spark-2.0.0/utils.scala

```diff
@@ -0,0 +1,19 @@
+package sparklyr
+
+import org.apache.spark.sql._
+
+object Utils200 {
```

Comment:
Yes, I think versioned directories is the right approach, we've been lucky so far to be able to workaround versioning issues in a single set of sources, but we were expecting this to happen at some point.